src/containers/App.js

```diff
@@ -0,0 +1,14 @@
+import '../styles/App.css';
+import BooksList from './BooksList';
+import BookForm from '../components/BookForm';
+
+function App() {
+  return (
+    <div className="App">
+      <BooksList />
+      <BookForm />
+    </div>
+  );
```

Comment:
- [ ] Also please move App.js to src/components because it is simply a presentational component.